### DISCUSSION TITLE
How can I set a default icon for each "state" succes, error, etc?

### DISCUSSION THREAD

**Discussion by viveleroi** [fkhadra/react-toastify#1019] [Q&A]
I want to provide a default icon for each state so that when my developers call `toast.success(...)` they don't have to pass the `{ icon }` config object. The library seems like it has no configuration for this.

**Comment by makibat** [comment]
Something like this should do the trick: 

 ```
<ToastContainer
  ...
  icon={({ type }) => {
    switch (type) {
      case 'success': {
        // return success icon
      }
      case 'error': {
        // return error icon
      }
    }
  }}
/>
```
